config/locales/en.yml

```diff
@@ -1656,7 +1656,7 @@ en:
     errors:
       code_401: "You need to be authenticated to access this resource."
       code_403: "You are not authorized to access this resource."
-      code_404: "Couldn\'t find %{type} with id=%{id}"
```

Comment:
That's because it used to be magically appended. See https://github.com/opf/openproject/pull/2555/files#diff-aa9a718d15c3f23754e451e96661d8ffL34 for that.